src/utils/db.js

```diff
@@ -0,0 +1,16 @@
+const { Sequelize } = require('sequelize');
+const { config } = require('dotenv');
+
+config();
+
+const client = new Sequelize({
+  host: process.env.DB_HOST,
+  username: process.env.DB_USER,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_DATABASE,
+  dialect: 'postgres',
+});
```

Comment:
The `Sequelize` constructor should be called with separate arguments for `database`, `username`, `password`, and an options object. Update the initialization to `new Sequelize(process.env.DB_DATABASE, process.env.DB_USER, process.env.DB_PASSWORD, { host: process.env.DB_HOST, dialect: 'postgres' });`.